pkg/tests/api/alerting/api_alertmanager_test.go

```diff
@@ -908,8 +908,9 @@ func TestIntegrationAlertRuleCRUD(t *testing.T) {
 						Annotations: map[string]string{"annotation1": "val1"},
 					},
 					GrafanaManagedAlert: &apimodels.PostableGrafanaRule{
-						Title: "AlwaysFiring",
-						Data:  []apimodels.AlertQuery{},
+						Title:     "AlwaysFiring",
+						Condition: "A",
+						Data:      []apimodels.AlertQuery{},
```

Comment:
The rule in this test was invalid in more than one way. By reordering validation checks, it caused this test to no longer cover what it was supposed to cover.

By fixing condition, we now cover the intent of this test a bit better, and it's not so tied to implementation.